### PR TITLE
Rt76734 - bug fix for target upload file radius being ignored in search results.

### PR DESCRIPTION
**Pull request by hjeeves**

### CODE REVIEW

File: caom2-search-lib/src/main/java/ca/nrc/cadc/search/upload/StreamingIterator.java [hunk 1]
```diff
@@ -134,6 +141,7 @@ public Element next() throws NoSuchElementException
         tableRow.addContent(createTableData(ra));
         tableRow.addContent(createTableData(dec));
         tableRow.addContent(createTableData(radius));
+        tableRow.addContent(createTableData(position));
```
Comment (at88mph) [inline caom2-search-lib/src/main/java/ca/nrc/cadc/search/upload/StreamingIterator.java:144]: Can the RA, Dec, and Radius be removed from the upload table now?  Or are we preserving them for the original values?

File: caom2-search-server/src/main/java/ca/nrc/cadc/caom2/ADQLGenerator.java [hunk 2]
```diff
@@ -378,6 +379,7 @@ String getSelectList(final String utypeSelectList) {
             // Upload target, ra and dec come in to the service as part of the
             // list of requested columns. Only radius needs to be added.
             sb.append(prefix + SEARCH_UPLOAD_TABLE + ".radius");
+            sb.append(prefix + SEARCH_UPLOAD_TABLE + ".position");
```
Comment (at88mph) [inline caom2-search-server/src/main/java/ca/nrc/cadc/caom2/ADQLGenerator.java:382]: Would you please fix lines 381 and 382 to be proper StringBuilder.append() calls?  Building the string inside the append defeats the purpose of it.  Thanks.

Also fix the comment above it.

**Review by at88mph — CHANGES_REQUESTED** [review]

File: caom2-search-lib/src/main/java/ca/nrc/cadc/search/upload/StreamingIterator.java (hunk 1 above)
Comment (hjeeves) [reply caom2-search-lib/src/main/java/ca/nrc/cadc/search/upload/StreamingIterator.java:144]: RA and Dec are still displayed in the results table, for both single target and target upload files, they are in the default columns displayed. 

The position strings when they return have 16 digit precision that adds a '2' on... I'm not sure if that's going to be appropriate for cutout use, if not then the radius' will be needed.

Comment (at88mph) [reply caom2-search-lib/src/main/java/ca/nrc/cadc/search/upload/StreamingIterator.java:144]: You're referring to the `Upload Target RA` and `Upload Target Dec`?  If that's the case and uses those columns then please leave them in.

File: caom2-search-server/src/main/java/ca/nrc/cadc/caom2/ADQLGenerator.java (hunk 2 above)
Comment (hjeeves) [reply caom2-search-server/src/main/java/ca/nrc/cadc/caom2/ADQLGenerator.java]: ok

File: caom2-search-lib/src/main/java/ca/nrc/cadc/search/upload/StreamingIterator.java (hunk 1 above)
Comment (hjeeves) [reply caom2-search-lib/src/main/java/ca/nrc/cadc/search/upload/StreamingIterator.java:144]: ok. didn't make any changes here.

**Review by at88mph — APPROVED** [review]

**Pull request merged**